### DISCUSSION TITLE
Add non-visible light all-sky survey(s) as backgrounds

### DISCUSSION THREAD

**Discussion by joeastro** [Stellarium/stellarium#2757] [Q&A] [answered]
Hello,

I am interested in adding an option to overlay an all-sky image from a non-visible light survey onto the sky in Stellarium. Functionally I am thinking of an overlay similar to the Milky Way Model that is currently used. The specific survey I had in mind is from the Fermi Gamma-Ray Space Telescope ([link](https://www.nasa.gov/feature/goddard/2016/nasas-fermi-space-telescope-sharpens-its-high-energy-vision)).

I'm new to Stellarium development so I'd greatly appreciate tips on where to get started. I'd also like to enable sources from the [Fermi Source Catalog](https://fermi.gsfc.nasa.gov/ssc/data/access/lat/10yr_catalog/) to appear at their sky positions as well (similar to how quasars are currently treated).

If it matters, I am a member of the Fermi Collaboration and think this could be a fantastic outreach tool, as well as be of interest to professional & amateur audiences alike.

Thank you for your time,
Joe

**Comment by gzotti** [comment, marked as answer]
Have you seen the HiPS surveys? See User Guide, chapter 10.
![stellarium-025](https://user-images.githubusercontent.com/18099873/196268718-4480f6eb-a973-4731-a111-8dd4e9c731f8.jpg)

For the source catalog, you could copy e.g. the Quasars plugin and adapt to the Fermi catalog.

**Reply by joeastro** [reply]
Thank you for the suggestions! The screenshot was most helpful, as I could not figure out how to get the different HiPS surveys to actually display.

Copying the quasar catalog was actually what I was thinking for doing the Fermi catalog, so it's good to know I was on the right track.
